Minor updates to access reviews(Beta) (#12973)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/accessreviewreviewer.md

---
title: "accessReviewReviewer resource type"

> description: "Represents a contacted reviewer for an access review."

Reviewer: ```suggestion
description: "Represents a reviewer who was contacted for an access review."
```

> author: "shubhamguptacal"

Reviewer: Isabelle's alias. Need [single point of contact ](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103185/GitHub-owners)for all customer issues relating to access reviews.

localization_priority: Normal
ms.prod: "governance"
doc_type: resourcePageType
---

> # accessReviewReviewer resource type

Reviewer 2: @FaithOmbongi does this resource type need to be added to the TOC?

Reviewer 2: And maybe we should add an explainer of the new "contacted reviewers" concept to [Assign reviewers to your access review definition using the Microsoft Graph API](/graph/accessreviews-scope-concept)

Reviewer: > does this resource type need to be added to the TOC?

No, shouldn't be exposed in the TOC.

> And maybe we should add an explainer of the new "contacted reviewers" concept to Assign reviewers to your access review definition using the Microsoft Graph API

Might be off because the article covers only accessReviewScheduleDefinition which doesn't have this property.

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

[!INCLUDE [accessreviews-disclaimer-v2](../../includes/accessreviews-disclaimer-v2.md)]

> In the access review process, reviewers(who are users) are contacted to do the review. Based on the reviewers configuration(groups, managers, etc.), the expanded list of users is constructed and those reviewers are contacted to do the review when the review starts. This navigation link returns a collection of those contacted reviewers. Note that this is different from the reviewers property above in the sense that the reviewers property stores the configuration of reviewers whereas the "contactedReviewers" propertys is returns the expanded collection of users based on that configuration. For example, one might configure owners of group G1[with owners U1,U2], members of group G2[with members U3,U4] and users U5,U6 in the "reviewers" property. The "contactedReviewers" property will return [U1,U2,U3,U4,U5,U6].

Reviewer 2: ```suggestion
This navigation link returns the collection of users who are contacted to complete a review, expanded to show individual users instead of reviewer types, such as group owner or manager. In the access review process, reviewers (who are users) are determined based on the configuration of the 'reviewers' and 'fallbackReviewers' properties of the [accessReviewScheduleDefinition](accessreviewscheduledefinition.md). From these configurations of [accessReviewScope](accessreviewscope.md) objects, the expanded list of users is constructed and those reviewers are contacted to do the review when the review starts.
```

> Inherits from [accessReviewScope](../resources/accessreviewscope.md).

Reviewer: ```suggestion
Inherits from [entity](entity.md).
```

## Properties
| Property | Type | Description |
| :-------------------------| :---------- | :---------- |
| id | String | Id of the reviewer |
| displayName | String | Name of reviewer |
| userPrincipalName | String | User principal name of the user |

> | createdDateTime | DateTimeOffset | Date when the reviewer was added |

Reviewer 2: @shubhamguptacal is it the date when the reviewer was added or contacted?

Reviewer: Can reviewers be added and contacted on different dates? Or do we capture dates of reminder notifications?
Proposed description
_The date when the reviewer was [added/contacted] for the access review._

Reviewer 2: Updated

> | id | String | Id of the reviewer |
> | displayName | String | Name of reviewer |
> | userPrincipalName | String | User principal name of the user |
> | createdDateTime | DateTimeOffset | Date when the reviewer was added |

Reviewer: ```suggestion
| id | String | Identifier of the reviewer. Inherited from [entity](entity.md). |
| displayName | String | Name of reviewer. |
| userPrincipalName | String | User principal name of the user. |
| createdDateTime | DateTimeOffset | Date when the reviewer was added. |
```

## Relationships
None.

## JSON representation
The following is a JSON representation of the resource.
<!-- {
  "blockType": "resource",
  "@odata.type": "microsoft.graph.accessReviewReviewer"
}
-->
``` json
{
  "@odata.type": "#microsoft.graph.accessReviewReviewer",
  "id": "String",
  "displayName": "String",
  "userPrincipalName": "String",

>   "createdDateTime": "DateTimeOffset"

Reviewer: ```suggestion
  "createdDateTime": "String (timestamp)",
```
